ingestion/tests/integration/ometa/test_ometa_messaging_service_api.py

```diff
@@ -0,0 +1,216 @@
+
+        self.assertEqual(res.name, self.entity.name)
+        self.assertEqual(res.serviceType, self.entity.serviceType)
+        self.assertEqual(
+            res.connection.config.hostPort, self.entity.connection.config.hostPort
+        )
+
+    def test_update_messaging_service(self):
+        """
+        Updating a Messaging Service entity changes its properties
+        """
+
+        original_res = self.metadata.create_or_update(data=self.create)
+
+        new_connection = MessagingConnection(
+            config=KafkaConnection(bootstrapServers="localhost:9092")
+        )
+
+        update_request = CreateMessagingServiceRequest(
+            name="test-messaging-service",
+            serviceType=MessagingServiceType.Kafka,
+            connection=new_connection,
+        )
+
+        updated_res: MessagingService = self.metadata.create_or_update(
+            data=update_request
+        )
+
+        # Same ID, updated owner
+        self.assertEqual(updated_res.id, original_res.id)
+        self.assertEqual(
+            updated_res.connection.config.hostPort, new_connection.config.hostPort
+        )
+
+    def test_get_name(self):
+        """
+        We can fetch a Messaging Service by name and get it back as Entity
+        """
+
+        self.metadata.create_or_update(data=self.create)
+
+        res = self.metadata.get_by_name(entity=MessagingService, fqn=self.entity.name)
+        self.assertEqual(res.name, self.entity.name)
+
+    def test_get_id(self):
+        """
+        We can fetch a Messaging by ID and get it back as Entity
+        """
+
+        self.metadata.create_or_update(data=self.create)
+
+        # First pick up by name
+        res_name = self.metadata.get_by_name(
+            entity=MessagingService, fqn=self.entity.name
+        )
+        # Then fetch by ID
+        res = self.metadata.get_by_id(entity=MessagingService, entity_id=res_name.id)
+
+        self.assertEqual(res_name.id, res.id)
+
+    def test_list(self):
+        """
+        We can list all our Messaging Services
+        """
+
+        self.metadata.create_or_update(data=self.create)
+
+        res = self.metadata.list_entities(entity=MessagingService)
+
+        assert self.entity.name in (ent.name for ent in res.entities)
+
+    def test_delete(self):
+        """
+        We can delete a Messaging by ID
+        """
+
+        self.metadata.create_or_update(data=self.create)
+
+        # Find by name
+        res_name = self.metadata.get_by_name(
+            entity=MessagingService, fqn=self.entity.name
+        )
+        # Then fetch by ID
+        res_id = self.metadata.get_by_id(
+            entity=MessagingService, entity_id=str(res_name.id.__root__)
+        )
+
+        # Delete
+        self.metadata.delete(
+            entity=MessagingService, entity_id=str(res_id.id.__root__), recursive=True
+        )
+
+        # Then we should not find it
+        res = self.metadata.list_entities(entity=Messaging)
```

Comment:
looks like the `Messaging` module which is imported incorrectly is only being used here, it looks like you are trying to delete the messaging service, and asserting if the entity got deleted in that case entity should be `MessagingService` itself right!